app/scripts/controllers/overview.js

```diff
@@ -1101,6 +1101,7 @@ function OverviewController($scope,
     // - API objects by binding name
     state.bindingsByApplicationUID = {};
     state.applicationsByBinding = {};
+    state.validBindingsByApplicationUID = {};
```

Comment:
This is specifically for the delete binding action, so I don't mind a name that makes that clear even if it doesn't roll off the tongue.

Alternatively we could put the filter in overview-list-row and call it `deleteableBindings`.